install/kubernetes/helm/istio/charts/kiali/templates/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: kiali
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: kiali
+spec:
+  type: NodePort
```

Comment:
We don't typically use NodePort - define a gateway +rule or ingress resource instead. 
NodePort can be used in some debug - but it should not be the default.

Reply:
Done, added an ingress first, will add `gateway +rule` in follow up PR, as this needs to be fixed for all other charts as well.